src/pybind/mgr/dashboard/frontend/src/styles/_carbon-defaults.scss

```
@@ -66,6 +70,54 @@ th {
   padding-block: 0 !important;
 }
 
+/******************************************
+ Tags
+******************************************/
+.tags-success {
```

A bit nitpicky but not a fan of plural names for this scss clasess

```suggestion
.tag-success {
```

---

Suggested changes have been made